src/card.css

```
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+  /* margin-top: 32px; */
+  margin-bottom: 40px;
+  /* margin-left: 19px;
+  margin-right: 19px; */
```

don't push comments please
```suggestion
  margin-bottom: 40px;
```